src/organizations/components/OrgProfileTab/DeletePaidOrgOverlay.tsx

```diff
@@ -0,0 +1,191 @@
+  const account = useSelector(selectCurrentAccount)
+  const org = useSelector(selectCurrentOrg)
+  const user = useSelector(selectUser)
+  const dispatch = useDispatch()
+
+  const {onClose} = useContext(OverlayContext)
+
+  const [deleteButtonStatus, setDeleteButtonStatus] = useState(
+    ComponentStatus.Disabled
+  )
+  const [userAcceptedTerms, setUserAcceptedTerms] = useState(false)
+
+  const toggleAcceptedTerms = () => {
+    const currentAcceptanceStatus = !userAcceptedTerms
+    setUserAcceptedTerms(currentAcceptanceStatus)
+
+    setDeleteButtonStatus(
+      currentAcceptanceStatus
+        ? ComponentStatus.Default
+        : ComponentStatus.Disabled
+    )
+  }
+
+  const handleDeleteOrg = async () => {
+    try {
+      setDeleteButtonStatus(ComponentStatus.Loading)
+
+      await deleteOrganization(org.id)
+
+      setDeleteButtonStatus(ComponentStatus.Default)
+      // Adding (and on reload, removing) this property from local storage
+      // is a work-around so that it's possible for the user to be notified of org deletion
+      // AFTER app reload. See src/components/notifications/Notifications.tsx,
+      // which detects the property, and if found, sends a notification then deletes it.
+      setToLocalStorage('justDeletedOrg', org.name)
+      window.location.href = `${CLOUD_URL}`
+
+      onClose()
+    } catch (err) {
+      dispatch(notify(deleteOrgFailed(org.name)))
+      reportErrorThroughHoneyBadger(err, {
+        name: 'Org deletion failed',
+        context: {
+          user,
+          account,
+          org,
+        },
+      })
+      setDeleteButtonStatus(ComponentStatus.Disabled)
+      setUserAcceptedTerms(false)
+    }
+  }
+
+  return (
+    <Overlay.Container
+      className="org-delete-overlay"
+      maxWidth={600}
+      testID="create-org-overlay--container"
+    >
+      <Overlay.Header
+        testID="create-org-overlay--header"
+        title="Delete Organization"
+        onDismiss={onClose}
+      />
+      <Overlay.Body>
+        <Alert
+          className="org-delete-overlay--warning-message"
+          color={ComponentColor.Danger}
+          icon={IconFont.AlertTriangle}
+        >
+          You will be able to recover this organization's data for up to 7 days
+          by contacting support. It will be unrecoverable afterwards.
+        </Alert>
+        <br />
+        <ul>
+          <li>
+            All of your writes, queries, and tasks for <b>{org.name}</b> will be{' '}
+            <b>suspended immediately</b>.
+          </li>
+          <li>
+            Your final billing statement will be calculated for any usage and
+            storage incurred prior to the deletion of your organization and your
+            credit card will be charged for the amount.
+          </li>
+          <li>
+            Before continuing, you are responsible for exporting any data or
+            content - including dashboards, tasks and variables - from the user
```

Comment:
```suggestion
            content - including dashboards, tasks, and variables - from the user
```
I think we want the Oxford comma here to distinguish tasks and variables as separate entities

Reply:
💯 Sure do.